index.js

```diff
@@ -235,6 +267,7 @@ function init (ssb, config) {
 
     application: scuttle.application,
     poBox: scuttle.poBox,
+    findBySubgroupId: scuttle.link.findParentGroupLinks,
```

Comment:
can we just call this method `findParentGroupLinks` because the current name  is unclear about what it's finding

Reply:
and actually, is this ... yeah I'm still confused